README.md

```
@@ -1,46 +1,138 @@
+Builds only the Firebase messaging service worker using esbuild.
 
 ### `yarn test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Launches the test runner in interactive watch mode.
 
-### `yarn build`
+### `yarn mock-server`
+
+Starts a mock API server using JSON Server at http://localhost:3001.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)